observability/logging/logging_5/log-visualization.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="log-visualization"]
+= About log visualization
+:context: log-visualization
+
+toc::[]
+
+You can visualize your log data in the {product-title} web console.
```

Comment:
Just trying to make clearer that this content is "outdated" even if using 5.8. WDYT
```suggestion
You can visualize your log data in the {product-title} web console. See the documentation about "Log visualization with the web console". This page contains instructions on how to configure the **Legacy** log visualizer based on Kibana, meant for use-cases still using ElasticSearch.
```

Reply:
updated with minor changes and also added the deprecation notice about  Elasticsearch Operator. WDYT?